### ISSUE TITLE
Training each cycle from scratch?

### ISSUE THREAD

**Issue by Ab-34** [yuantn/MI-AOD#102]
Thank you very much for your code!

I have a small doubt.

Are you training each model from scratch for each cycle? 
As for each cycle, a new model is being initialized in line 131. And I don't see any checkpoints being loaded after that.

**Comment by yuantn** [comment]
Yes, you can debug deeply. It involves calls the function mmcv.utils.registry.build_from_cfg, which loads the parameters in cfg.model and builds the model from it. The path of the pretrained model is set in cfg.model (torchvision://resnet50), so for each cycle it is trained from scratch.
